scripts/openstack-quickstart-demosetup

```diff
@@ -466,7 +470,7 @@ done
 ## setup loopback LVM configuration
 #-----------------------------------------
 
-
+start_and_enable_service lvm2-lvmetad
```

Comment:
```start_and_enable_service``` is not avaialble in the loopback-lvm script. That's why I thought it's simpler that way.